docs/specifications/tests/Connectivity-TP/connectivity01.md

```diff
@@ -1,39 +1,217 @@
-## CONNECTIVITY01: UDP connectivity
+# CONNECTIVITY01: UDP connectivity to name servers
 
-### Test case identifier
+## Test case identifier
+**CONNECTIVITY01**
 
-**CONNECTIVITY01:** UDP connectivity
 
-### Objective
+## Table of contents
 
-DNS queries are sent using UDP on port 53, as described in section 4.2.1 of
-[RFC 1035](https://tools.ietf.org/html/rfc1035).
+* [Objective](#objective)
+* [Scope](#scope)
+* [Inputs](#inputs)
+* [Summary](#summary)
+* [Test procedure](#test-procedure)
+* [Outcome(s)](#outcomes)
+* [Special procedural requirements](#special-procedural-requirements)
+* [Intercase dependencies](#intercase-dependencies)
+* [Terminology](#terminology)
 
-The objective for this test is that all the authoritative name servers for
-the domain are accessible over UDP on port 53
 
-### Inputs
+## Objective
 
-The domain to be tested.
+UDP is the fundamental protocol to reach a general purpose name server hosting
+a zone, "DNS servers MUST be able to service UDP [...]" ([RFC 1123], section
+6.1.3.2, page 75), also restated in [RFC 7766][RFC 7766#5], section 5.
 
-### Ordered description of steps to be taken to execute the test case
+This test case will verify if the name servers in the delegation of and zone of
```

Comment:
Capitalize "test case" everywhere in this section.

Reply:
Updated.